data/card_catalogue/unique-1.json

```diff
@@ -179,36 +129,6 @@
       "image": "assets/core/card/illustration/Prison.png"
     }
   },
-  {
-    "title": "The Swarm",
-    "upgrade": "",
-    "tier": "green",
-    "details": {
-      "rules": "Pick a small animal or insect for your target to be swarmed by.",
-      "flavour": "The sky went dark, and all sound was drowned out by the hum of their arrival.",
-      "image": "assets/core/card/illustration/The Swarm.png"
-    }
-  },
-  {
-    "title": "Glass Cannon",
-    "upgrade": "",
-    "tier": "red",
-    "details": {
-      "rules": "Until your next turn, deal double damage and take double damage.",
-      "flavour": "\"[...] the stakes, therefore, remain at the same altitude\" - Dr. F. Mickleson, Noted Polymath",
-      "image": "assets/core/card/illustration/Glass Cannon.png"
-    }
-  },
-  {
-    "title": "Monster Deterrent",
-    "upgrade": "",
-    "tier": "blue",
-    "details": {
-      "rules": "Creates a strong odor hardly noticeable by humans but detested by all other lifeforms.",
-      "flavour": "He who smelt it was dealt it.",
-      "image": "assets/core/card/illustration/Monster Deterrent.png"
-    }
-  },
```

Comment:
Not going to be many monsters in the upcoming campaign, this was just too niche and not particularly fun when it whiffed

Reply:
I'd be up for re-introducing blues like this one en masse once we've amassed a bunch of them. For loot or whatever. Maybe with the clause that you don't pull blues/these in particular during character creation.

Reply:
I think they'd be great as NPC drops, which I haven't been including in the loot decks. Then you pick them up during the session where they're most relevant